server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
@adrienpeiffer Semantic versioning dictates that you reset the numbers on the right when a number on the left of it has changed. This jives with that IMO